packet/flatcar-linux/kubernetes/cl/controller-install.yaml.tmpl

```diff
@@ -5,10 +5,14 @@ systemd:
       enable: true
       contents: |
         [Unit]
+        StartLimitBurst=5
+        StartLimitIntervalSec=3600s
```

Comment:
Interesting. So `StartLimitIntervalSec` requires a `unit` along with the number but `RestartSec` doesn't.
[Docs for `RestartSec`](https://www.freedesktop.org/software/systemd/man/systemd.service.html#RestartSec=) quotes:

>  Takes a unit-less value in seconds, or a time span value such as "5min 20s". 

But I couldn't find anything for `StartLimitIntervalSec` [here](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#StartLimitIntervalSec=interval).

Reply:
When I specified it without `s` at the end, it didn't pick it up and it fallback to 1m 30s default IIRC.